### PR TITLE
Initialize project

### PR DESCRIPTION
**Pull request by DanielSiepmann**
Provide first implementation with basic setup.

### CODE REVIEW

File: Classes/Form/FormElement/FileCollectionElement.php [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+            $this->setProperty('options', $this->getOptions($value));
+            return;
+        }
+
+        parent::setProperty($key, $value);
+    }
+
+    /**
+     * @param array<string, string> $configuration
+     * @return array<string, string>
```
Comment (justusmoroni) [inline Classes/Form/FormElement/FileCollectionElement.php:49]:
```suggestion

     * @return array<string, string>
```

It's like this in line 76-78.

File: README.rst [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+TYPO3 Extension Form File Collection
+====================================
+
+Adds new EXT:form element used to render a file collection allowing the visitor to choose files.
+
+Concept
+=======
+
+The form element will fetch the configured file collection and assign proper options based on the contained files.
+That allows for using existing templates that allow to select one or multiple options.
```
Comment (justusmoroni) [inline README.rst]: Sounds weird. Maybe:

```suggestion
That allows usage of existing templates that already handle one or multiple options.
```

File: Tests/Functional/FormIntegrationTest.php [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+use TYPO3\TestingFramework\Core\Functional\FunctionalTestCase;
+
+class FormIntegrationTest extends FunctionalTestCase
+{
+    use TestingFramework;
+
+    public function setUp(): void
+    {
+        $this->coreExtensionsToLoad = [
+            'fluid_styled_content',
```
Comment (justusmoroni) [inline Tests/Functional/FormIntegrationTest.php:37]: Is this used? We don't require it in the test extension.

File: phpunit.xml.dist [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+    convertErrorsToExceptions="true"
+    convertWarningsToExceptions="true"
+    forceCoversAnnotation="false"
+    processIsolation="false"
+    stopOnError="false"
+    stopOnFailure="false"
+    stopOnIncomplete="false"
+    stopOnSkipped="false"
+    verbose="false"
+    >
```
Comment (justusmoroni) [inline phpunit.xml.dist]:
```suggestion
>
```

**Review by justusmoroni — CHANGES_REQUESTED** [review]

File: Tests/Functional/FormIntegrationTest.php (hunk 3 above)
Comment (DanielSiepmann) [reply Tests/Functional/FormIntegrationTest.php:37]: We do, we render the content, and we would not have any rendering definition for CType `list`, that's provided by `fluid_styled_content`.

File: Classes/Form/FormElement/FileCollectionElement.php (hunk 1 above)
Comment (DanielSiepmann) [reply Classes/Form/FormElement/FileCollectionElement.php:49]: Added cs fixer rule and applied it.

**Review by justusmoroni — APPROVED** [review]

**Pull request merged**